docs/cli-reference/dfx-canister.mdx

```diff
@@ -1137,14 +1138,17 @@ You can specify the following options for the `dfx canister update-settings` com
 |-------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `--add-controller <principal>`            | Add a principal to the list of controllers of the canister.                                                                                                                                                                                                                                                                                                                              |
+| `--add-log-viewer <principal>`            | Add a principal to the list of log viewers of the canister.                                                                                                                                                                                                                                                                                                                              |
 | `-c`, `--compute-allocation <allocation>` | Specifies the canister's compute allocation. This should be a percent in the range [0..100].                                                                                                                                                                                                                                                                                             |
 | `--confirm-very-long-freezing-threshold`  | Freezing thresholds above ~1.5 years require this option as confirmation.                                                                                                                                                                                                                                                                                                                |
 | `--set-controller <principal>`            | Specifies the identity name or the principal of the new controller. Can be specified more than once, indicating the canister will have multiple controllers. If any controllers are set with this parameter, any other controllers will be removed.                                                                                                                                      |
+| `--set-log-viewer <principal>`            | Specifies the the principal of the log viewer of the canister. Can be specified more than once, indicating the canister will have multiple log viewers. If any log viewers are set with this parameter, any other log viewers will be removed.                                                                                                                                           |
```

Comment:
Unlike `--set-controller` a new option `--set-log-viewer` also changes the setting of log visibility (if it was `public` or `controllers`). This message has to clarify it.

Reply:
Fixed at https://github.com/dfinity/sdk/pull/3921/commits/201e2b727e6f000da193c446a0d68d4914eb3b0c.